src/main/java/com/mewna/catnip/entity/impl/EntityBuilder.java

```diff
@@ -1592,7 +1592,8 @@ public ApplicationCommandInteractionDataOption createApplicationCommandInteracti
                 .catnip(catnip)
                 .name(data.getString("name"))
                 .options(toList(data.getArray("options"), this::createApplicationCommandInteractionDataOption))
-                .value(ApplicationCommandOptionType.byKey(data.getInt("value")))
+                .type(ApplicationCommandOptionType.byKey(data.getInt("type")))
+                .value(data.get("value"))
```

Comment:
Deserialise to right type

Reply:
Always String so making value type String. Maybe fetching role, user, channel data from the provided id in value in the future.